Matrix_multiplication/dense.temper.md

```diff
@@ -0,0 +1,128 @@
+# Dense matrix math
```

Comment:
This file is a modified copy from the previously existing array-per-row matrix implementation. The two files can be diffed for comparison.